front/components/assistant/AssistantPicker.tsx

```diff
@@ -98,28 +99,32 @@ export function AssistantPicker({
               </>
             }
             bottomBar={
-              <div className="flex border-t border-structure-50 p-2">
-                <Link
-                  href={`/w/${owner.sId}/builder/assistants/new?flow=personal_assistants`}
-                >
-                  <Button
-                    label="Create"
-                    size="xs"
-                    variant="primary"
-                    icon={PlusIcon}
-                    className="mr-2"
-                  />
-                </Link>
-                <div className="s-flex-grow" />
-                <Link href={`/w/${owner.sId}/assistant/assistants`}>
-                  <Button
-                    label="My Assistants"
-                    size="xs"
-                    variant="tertiary"
-                    icon={ListIcon}
-                  />
-                </Link>
-              </div>
+              <>
```

Comment:
It feels weird to have both transcript in user menu and sub navigation, can we pick only one?

Reply:
It's only in user menu, subnav leftover must have been from all that PR splitting :D